trunk.yaml

```
@@ -0,0 +1,9 @@
+# Trunk configuration - see https://docs.trunk.io/check/configuration
+# Run `trunk init` locally to add or update linters.
+version: "0.1"
+
+lint:
+  enabled:
+    - eslint
+    - prettier
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)